[0m16:47:02.635017 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11136f5d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x111385390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x111385d50>]}


============================== 16:47:02.638318 | 8e8f9029-cb01-4159-ad7e-5acfcf804773 ==============================
[0m16:47:02.638318 [info ] [MainThread]: Running with dbt=1.6.0
[0m16:47:02.638741 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': '/Users/joshua/.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt init whostat', 'send_anonymous_usage_stats': 'True'}
[0m16:47:02.639481 [debug] [MainThread]: Starter project path: /Users/joshua/.virtualenvs/whostat/lib/python3.11/site-packages/dbt/include/starter_project
[0m16:47:02.644350 [info ] [MainThread]: 
Your new dbt project "whostat" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:47:02.644637 [info ] [MainThread]: Setting up your profile.
[0m16:57:04.923116 [info ] [MainThread]: Profile whostat written to /Users/joshua/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:57:04.926038 [debug] [MainThread]: Command `dbt init` succeeded at 16:57:04.924900 after 602.33 seconds
[0m16:57:04.926874 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1113858d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11136d910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10b27bf10>]}
[0m16:57:04.927516 [debug] [MainThread]: Flushing usage events
[0m17:06:45.413567 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112e93710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112eb96d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112eb9dd0>]}


============================== 17:06:45.416921 | ceba1525-22c4-4e42-bf4c-db8b37a10735 ==============================
[0m17:06:45.416921 [info ] [MainThread]: Running with dbt=1.6.0
[0m17:06:45.417357 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joshua/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'invocation_command': 'dbt compile', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m17:06:45.417790 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joshua/Documents/personal-projects/whostat/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m17:06:45.418623 [debug] [MainThread]: Command `dbt compile` failed at 17:06:45.418263 after 0.04 seconds
[0m17:06:45.418945 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112eba190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112e91310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10cdd7e10>]}
[0m17:06:45.419249 [debug] [MainThread]: Flushing usage events
[0m17:11:37.493364 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110be3450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110c0d390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110c0e010>]}


============================== 17:11:37.496981 | 5c630f00-51ca-4359-a8a8-f488c7c12015 ==============================
[0m17:11:37.496981 [info ] [MainThread]: Running with dbt=1.6.0
[0m17:11:37.497467 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/joshua/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt run', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m17:11:37.497904 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joshua/Documents/personal-projects/whostat/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m17:11:37.498579 [debug] [MainThread]: Command `dbt run` failed at 17:11:37.498292 after 0.04 seconds
[0m17:11:37.498866 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110be15d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110bec310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10f9f7a50>]}
[0m17:11:37.499108 [debug] [MainThread]: Flushing usage events
[0m17:15:03.123677 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110690c10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110dc9890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110dc9f90>]}


============================== 17:15:03.127098 | e3f2d589-48a2-4dcc-97b4-7349eec70413 ==============================
[0m17:15:03.127098 [info ] [MainThread]: Running with dbt=1.6.0
[0m17:15:03.127547 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joshua/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt compile', 'send_anonymous_usage_stats': 'True'}
[0m17:15:03.128017 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joshua/Documents/personal-projects/whostatdbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m17:15:03.128722 [debug] [MainThread]: Command `dbt compile` failed at 17:15:03.128436 after 0.03 seconds
[0m17:15:03.129018 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110da1390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x110dac4d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10acdfe50>]}
[0m17:15:03.129285 [debug] [MainThread]: Flushing usage events
[0m17:15:38.751607 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11381fd10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x113845c10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x113846310>]}


============================== 17:15:38.754371 | 0146b0cd-6513-48f1-b2de-c79a53440897 ==============================
[0m17:15:38.754371 [info ] [MainThread]: Running with dbt=1.6.0
[0m17:15:38.754811 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/Users/joshua/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt compile', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m17:15:38.755282 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/joshua/Documents/personal-projects/whostatdbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m17:15:38.756092 [debug] [MainThread]: Command `dbt compile` failed at 17:15:38.755752 after 0.04 seconds
[0m17:15:38.756439 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x11381d650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1138244d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x113812f90>]}
[0m17:15:38.756752 [debug] [MainThread]: Flushing usage events
